index.pug

```diff
@@ -0,0 +1,591 @@
+head
```

Comment:
Please add title and desc

ex:
---
title: Create Custom Branded UPI/Bharat QR code with Razorpay MultiQR
description: Unlimited UPI/Bharat QR codes for merchants. Put your Business Name, Logo on QR codes. Accept digital payments from customers using UPI apps, wallets, cards directly.
---

Reply:
![Screenshot 2021-12-03 at 7 11 32 PM](https://user-images.githubusercontent.com/10862593/144612033-72270816-c700-4113-aab6-75489eb8803c.png)